src/gui/pedalboard/editors/KnobsComponent.cpp

```diff
@@ -312,6 +312,36 @@ void KnobsComponent::resized()
         return;
     }
 
+    if (totalNumComponents == 5 && sliders.size() == 5)
+    {
+        int compIdx = 0;
+
+        Rectangle<int> bounds[5];
+        const int sWidth = proportionOfWidth (0.215f);
```

Comment:
Could these be more exact?

Reply:
These seem fine to me. At least when I zoom in/out I don't see any noticeable movement that you might get from rounding errors or something like that.